doc-content/enterprise-only/springboot/spring-boot-jms-audit-proc.adoc

```diff
@@ -0,0 +1,126 @@
+
+NOTE: Only audit data is stored in the message broker. No other data is replicated.
+
+.Prerequisites
+* You have a {PRODUCT} Spring Boot application.
+
+.Procedure
+. Open the Spring Boot application's `pom.xml` file in a text editor.
+. Add the {KIE_SERVER} Spring Boot audit dependency to the `pom.xml` file:
++
+[source, xml]
+----
+<dependency>
+  <groupId>org.kie</groupId>
+  <artifactId>kie-server-spring-boot-autoconfiguration-audit-replication</artifactId>
+  <version>${version.org.kie}</version>
+</dependency>
+----
+. Add the dependency for your JMS client. The following example adds the Advanced Message Queuing Protocal (AMQP) dependency:
++
+[source, xml]
+----
+<dependency>
+			<groupId>org.amqphub.spring</groupId>
+			<artifactId>amqp-10-jms-spring-boot-starter</artifactId>
+			<version>2.2.6</version>
+		</dependency>
+----
+. Add the JMS pool dependency:
++
+[source, xml]
+----
+<dependency>
+  <groupId>org.messaginghub</groupId>
+  <artifactId>pooled-jms</artifactId>
+</dependency>
+----
+. To configure {KIE_SERVER} audit replication to use queues, complete the following tasks:
++
+.. Add the following lines to your Spring Boot application's `application.properties` file:
++
+[source]
+----
+kieserver.audit-replication.producer=true
+kieserver.audit-replication.queue=audit-queue
+----
+.. Add the properties required for your message broker client. The following example shows how to configure {KIE_SERVER} for AMPQ where `<JMS_HOST_PORT>` is the port that the broker listens on and  `<USERNAME>` and `<PASSWORD` are the login credentials for the broker:
++
+[source]
+----
+amqphub.amqp10jms.remote-url=<JMS_HOST_PORT>
+amqphub.amqp10jms.username=<USERNAME>
+amqphub.amqp10jms.password=<PASSWORD>
+amqphub.amqp10jms.pool.enabled=true
+----
+
+.. Add the following lines to the `application.properties` file of the service that will consume the message broker data:
++
+[source]
+----
+kieserver.audit-replication.consumer=true
+kieserver.audit-replication.queue=audit-queue
+----
+.. Add the properties required for your message broker client to the `application.properties` file of the service that will consume the message broker data. The following example shows how to configure {KIE_SERVER} for AMPQ where  `<JMS_HOST_PORT>` is the port that your message broker listens on and `<USERNAME>` and `<PASSWORD>` are the login credentials for the message broker:
++
+[source]
+----
+amqphub.amqp10jms.remote-url=amqp://<JMS_HOST_PORT>
+amqphub.amqp10jms.username=<USERNAME>
+amqphub.amqp10jms.password=<PASSWORD>
+amqphub.amqp10jms.pool.enabled=true
+----
+. To configure {KIE_SERVER} audit replication to use topics, complete the following tasks:
```

Comment:
Same as earlier comment.

Configure {KIE_SERVER} audit replication to use topics.